python/pylibraft/pylibraft/test/test_distance.py

```diff
@@ -64,9 +64,10 @@ def test_distance(n_rows, n_cols, inplace, metric, order, dtype):
     input1_device = device_ndarray(input1)
     output_device = device_ndarray(output) if inplace else None
 
-    handle = Handle()
+    s2 = Stream()
+    handle = Handle(stream=s2)
```

Comment:
why is this change necessary for this particular test? Not sure if it adds something now that there is a `test_handle`

Reply:
This one is mostly there just to verify that using an existing stream doesn't cause any crazy errors to the output of the function. I only added it to this test because it doesn't really seem like it needs to be in all the tests and i didn't want to pollute the simplicity of the handle tests by re-writing the pairwise distance test in its entirety. We could remove it, but it seems pretty benign (and it's also helpful).